_rules/block-collapsible-3e12e1.md

```diff
@@ -29,10 +29,10 @@ This rule applies to any [HTML web page][].
 
 ## Expectation
 
-For each [block of repeated content][] in each test target, which is before (in the [flat tree][]) at least one node of [non-repeated content after repeated content][], all the following are true:
+For each test target, and each of its [blocks of repeated content][block of repeated content] which are before at least one node of [non-repeated content after repeated content][] in the [flat tree][], all the following are true:
 
-- there exists an [instrument][] to make all nodes in this [block][] not [visible][]; and
-- there exists an [instrument][] to remove all nodes in this [block][] from the [accessibility tree][included in the accessibility tree].
+- there exists an [instrument][] before the [block][] to make all nodes in this [block][] not [visible][]; and
+- there exists an [instrument][] before the [block][] to remove all nodes in this [block][] from the [accessibility tree][included in the accessibility tree].
```

Comment:
"before" is a bit ambiguous here. Is that visually above, is that the relation in the flat tree? Is that the accessibility tree (since with aria-owns that can differ from the flat tree).